src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterService {
+    void writeToFile(String toFileName, String data);
```

Comment:
Consider adding documentation to the interface and its method to provide context and guidance for future developers.